users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -237,6 +253,40 @@ def get_returnn_config(
         # network["source"] = specaug_layer_jingjing(in_layer=["features"])
         pass
 
+    if audio_perturbation and recognition:
+        if preemphasis_perturbation:
+            for layer in feature_net["subnetwork"]:
+                layer_config = feature_net["subnetwork"][layer]
+                if layer_config.get("class") != "variable":
+                    if feature_net["subnetwork"][layer].get("from") == "data":
+                        feature_net["subnetwork"][layer]["from"] = "preemphasis"
```

Comment:
Was the check for `"variable"` just so `.get("from")` would not crash?
```suggestion
                if feature_net["subnetwork"][layer].get("from", None) == "data":
                    feature_net["subnetwork"][layer]["from"] = "preemphasis"
```